content/ko/docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -0,0 +1,238 @@
+---
+title: 데몬셋
+content_template: templates/concept
+weight: 50
+---
+
+{{% capture overview %}}
+
+_데몬셋_ 은 모든(또는 일부) 노드가 파드의 사본을 실행하도록 한다. 노드가 클러스터에 추가되면 
+파드도 추가된다. 노드가 클러스터에서 제거되면 해당 파드는 가비지로 
+수집된다. 데몬셋을 삭제하면 데몬셋이 생성한 파드들이 정리된다.
```

Comment:
```suggestion
수집된다. DaemonSet을 삭제하면 DaemonSet이 생성한 파드들이 정리된다.
```

Reply:
and I recommend a similar change elsewhere.

Reply:
Same as the previous comment.